contrib/mesos/pkg/election/etcd_master_test.go

```diff
@@ -58,41 +57,77 @@ func TestEtcdMasterNoOther(t *testing.T) {
 	w.Stop()
 }
 
+// MockClient is wrapper aroung tools.EtcdClient.
+type MockClient struct {
+	client tools.EtcdClient
```

Comment:
There is dependency from tools.EtcdClient in the implementation (non-test code). So this is just for consistency.